### ISSUE TITLE
Unable to unlock phone or pause audio when using Talkback 3.5.2 beta 2 and phone with passcode.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#387]
```
To reproduce the problem.
1. Start a track playing using Spotify.
2. Go to the home screen while the track is playing.
3. Lock the phone.
4. With passcode enabled attempt to unlock the phone. 

With Talkback 3.5.1 I am able to access the keypad to unlock the phone. I am 
also able to stop playback of the spotify track from the lock screen widget. 
With talkback 3.5.2 beta 2 when attempting to unlock the phone Talkback 
continuously announces "status widget" and "null widget". This makes it 
impossible to either unlock the phone or even hit the play/pause button on the 
lock screen to stop the track playing on Spotify.

I am using Talkback 3.5.2 beta 2 on a Nexus 4 running Android 4.4.4Please 
```

Original issue reported on code.google.com by `jared.st...@gmail.com` on 25 Aug 2014 at 6:52
